server/helpers/relayer.ts

```diff
@@ -1,5 +1,11 @@
 import { Wallet } from '@ethersproject/wallet';
 
+declare var process : {
```

Comment:
Any reason we need this here but not on others place where we use `process.env...`? Can't this be handled with the config files so we don't need to add types everytime?

Reply:
@bonustrack removed it.. it is interpreting the value as undefined ..